website/pages/component/tabs/example.tsx

```diff
@@ -75,3 +75,329 @@ export default function TabsLiveExample() {
     </LiveExample>
   );
 }
+
+// import React, { useState } from 'react';
```

Comment:
This was the example that I created with @sophstad who initially raised the bug. Wanted to leave the code so that you can run locally and check as well, but I will remove before we merge.